agent/util-scripts/gold/pbench-results-move/test-23.txt

```diff
@@ -7,11 +7,11 @@ Options:
   -C, --config PATH              Path to a pbench-agent configuration file
                                  (defaults to the '_PBENCH_AGENT_CONFIG'
                                  environment variable, if defined)  [required]
-  -a, --access [public|private]  pbench tarball access permission
-                                 public/private  [default: private]
-  --controller TEXT              Override the default controller name
```

Comment:
I wonder if `click` has an option to emit the help texted "sorted" somehow?

Reply:
Yeah, I was grmping about that, too.

Reply:
Since 2016: https://github.com/pallets/click/issues/672 😆 

The conversation does suggest a couple of possible workarounds, though.

Reply:
OK, I think I have a solution for this implemented, now.